index.js

```diff
@@ -96,6 +107,50 @@ function parse(str, options) {
  */
 
 function serialize(name, val, options) {
+  if( typeof name === 'object') {
+    var cookies = name;
+    var serializeOptions = val;
+
+    var cookieNames = Object.keys(cookies);
+    if(0 === cookieNames.length) {
+      return '';
+    } else if(cookieNames.length > 1) {
+      // If there are more than one cookies to serialize, only allow
+      // an encoding option to be set
+      var opt = serializeOptions || {};
+      serializeOptions = {
```

Comment:
I see, so if we are given multiple cookies to serialize, would it be best to return an array of serialized cookie strings rather than a single string with all cookies?